x-pack/plugins/security_solution/public/common/components/top_n/top_n.tsx

```diff
@@ -159,6 +152,13 @@ const TopNComponent: React.FC<Props> = ({
           />
         )}
       </TopNContent>
+
+      <CloseButton
```

Comment:
What is the reason for this change? Was the button hidden behind another element?

Reply:
@machadoum exactly.. See this screenshot from kibana dev in 8.9. So simply changing the DOM order resolves this issue. I should have added the video for that as well but missed it.

![image](https://github.com/elastic/kibana/assets/7485038/cc0568d8-5a9a-491d-846d-de714e28c8f1)